docs/topics/cache.txt

```diff
@@ -803,6 +803,30 @@ If you need to know whether ``add()`` stored a value in the cache, you can
 check the return value. It will return ``True`` if the value was stored,
 ``False`` otherwise.
 
+In cases where you may wish to use either a cached value or create a new cached
```

Comment:
If you want to get a key's value or set a value if the key isn't in the cache, there is the...